README.md

```diff
@@ -1,24 +1,175 @@
+
+- [Live Demo Link](https://rails-budget-app-5jzx.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- WALKTHROUGH VIDEO -->
+
+## Walkthrough video <a name ="walkthrough-video"></a>
+
+[Walkthrough video](https://www.loom.com/share/e29e20ee84774078b6791d108c7d9630)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prequisites
+
+In order to run this project you need:
+
+- To install ruby on rails on your PC. You can read [this](https://guides.rubyonrails.org/) documentation on how to do so
+- Know how to navigate directories or folders at the CLI.
+- Know how to get the URL(https/ssh) of a repository on GitHub.
+- You should have a code editor installed, preferably VSCode.
+
+### Setup
+
+In desired folder or directory in the CLI, run the command:
+
+```JavaScipt
+cd clone https://github.com/RoyJumah/Budgetify.git
+```
+
+Navigate into the cloned folder or repository by running the command:
+
+```JavaScript
+ cd <budgetify_app>
+```
+
+If VsCode is your default code editor, run:
+
+```JavaScript
+  code .
+```
+
+To install all the dependencies, run:
+
+```JavaScript
+ bundle install
+```
+After installing all the dependencies with the bundle install command, run the command `rails s` to start the server, and then navigate to http://localhost:3000 in your web browser to access the application.
+
+You are all set up!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  $ bundle exec rspec
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Roy Jumah**
+
+- GitHub: [@RoyJumah](https://github.com/RoyJumah)
+- Twitter: [@\_royissues](https://twitter.com/_royissues)
+- LinkedIn: [Roy Jumah](https://www.linkedin.com/in/roy-jumah/)
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] Plan budget effectively
+- [ ] Visualize expense data
+- [ ] Set payment reminders
+- [ ] Support multiple currencies
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/RoyJumah/Budgetify/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can give it a star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+We would like to thank `Microverse` for providing us the resources that enabled us to finish this challenge!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=the%20author.%20Therefore%2C-,you%20must%20do%20it%20in%20the%20README%20of%20your%20project.,-Interactions), Kindly give credits to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) at `Acknowledgments` section.

Reply:
Noted, I will work on this